Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Export_Single_Asset_to_SIEM_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,33 @@
+Playbook to handle incident triggered from PANW Iot (Zingbox) UI to send sing Alert or Vulnerability to SIEM.
```

Comment:
This playbook handles a single incident created in the PANW IoT (Zingbox) UI, creates a corresponding incident type in Cortex XSOAR, and sends the alert or vulnerability to your SIEM.  The incident types created in Cortex XSOAR are one of the following:
- Alert: PANW IoT 3rd Party SIEM Integration - Alert 
- Vulnerability: PANW IoT 3rd Party SIEM Integration - Vulnerability